projects/1.20/assets/skays-overworld-netherite-ore/overworld_netherite_ore/lang/zh_cn.json

```
@@ -0,0 +1,25 @@
+{
+  "advancements.master_of_ancient_shards.descr": "获得一组大型远古残骸碎片.",
```

```suggestion
  "advancements.master_of_ancient_shards.descr": "获得一组大型远古残骸碎片。",
```

标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。